### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by GitHubEason**
In this version, Quickstart, Features, Commands, Environment Variables, API Documentation, and Logging is added.

### CODE REVIEW

**Review by LawrenceLiao — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,10 +1,81 @@
...
+- **Editor config**: consistent editor configuration using [EditorConfig](https://editorconfig.org)
+
+## Commands
+
+Running locally:
+
+```bash
+yarn dev
+```
+
```
Comment (LawrenceLiao) [inline README.md:49]: You might be missing nodejs version as a prerequisite here.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,81 @@
...
-PORT =
-API_PREFIX =
 
-# needs to be added
---|test
+## Table of Contents
+
+- [CourtCanva service](#courtcanva-service)
+  - [Quick Start](#quick-start)
+  - [Table of Contents](#table-of-contents)
```
Comment (LawrenceLiao) [inline README.md:22]: Don't put unused links here to cause confusion.

**Review by chui-mel — APPROVED** [review]
LGTM

**Comment by stevenliu986** [comment]
Is the dependency manager yarn or npm? Cause I saw npm is in package.json's scripts.

**Review by Fuchih — APPROVED** [review]
looks good to me

**Pull request merged**